web/app/static/tags/network/azurenetworkform.tag

```diff
@@ -134,7 +134,7 @@
                         <label class="col-sm-3 control-label" for="azure_networkSubnetId">{{msg.network_azure_form_subnet_id_label}}</label>
 
                         <div class="col-sm-9">
-                            <input type="text" class="form-control" name="azure_networkSubnetId" ng-model="network.parameters.subnetId" id="azure_networkSubnetId" placeholder="{{msg.network_azure_form_subnet_id_placeholder}}" ng-pattern="/^[-a-zA-Z0-9]*$/" required>
+                            <input type="text" class="form-control" name="azure_networkSubnetId" ng-model="network.parameters.subnetId" id="azure_networkSubnetId" placeholder="{{msg.network_azure_form_subnet_id_placeholder}}" required>
                             <div class="help-block" ng-show="azureNetworkForm_2.azure_networkSubnetId.$dirty && azureNetworkForm_2.azure_networkSubnetId.$invalid">
                                 <i class="fa fa-warning"></i> {{msg.network_subnetid2_invalid}}
                             </div>
```

Comment:
I think we shouldn't remove these validators completely, but align them to the azure naming conventions:
resource groups:
<img width="519" alt="screen shot 2016-06-09 at 9 49 35 am" src="https://cloud.githubusercontent.com/assets/1568469/15922390/a248e902-2e27-11e6-8125-ae38ffa5f981.png">

virtual networks/subnets:
<img width="504" alt="screen shot 2016-06-09 at 9 50 30 am" src="https://cloud.githubusercontent.com/assets/1568469/15922397/ad0aaf1a-2e27-11e6-85d3-99949336d25d.png">